.travis.yml

```diff
@@ -52,9 +52,9 @@ after_success:
 jobs:
   include:
     - stage: "Documentation"
-      julia: 1.0
+      julia: 1.1
```

Comment:
You don't have to change it in this PR but I believe if you use `julia: 1` it'll always use the latest stable version. If you want to ensure the documentation is always built on the latest stable version you can do that. Typically, I fix every version for something like documentation so that I can ensure it's always possible to build in the future, i.e. pin dependencies in Project.toml, use a fixed version for Julia etc, even if it's an older version of the software.

Reply:
ok. I will make this change.